modules/schema/src/main/scala/lucuma/odb/json/gmos.scala

```diff
@@ -208,10 +208,12 @@ trait GmosCodec {
     Encoder.instance[GmosFpuMask[A]] {
       case GmosFpuMask.Builtin(v)       =>
         Json.obj(
-          "builtin"    -> v.asJson
+          "builtin"    -> v.asJson,
+          "customMask" -> Json.Null
```

Comment:
I remain unconvinced that Grackle should attempt to find a database mapping when processing JSON and encountering missing optional fields.